airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/source.py

```diff
@@ -69,6 +70,10 @@ class SourceZendeskSupport(AbstractSource):
     support and sales software which aims for quick implementation and adaptation at scale.
     """
 
+    @classmethod
+    def get_default_start_date(cls):
+        return pendulum.now(tz="UTC").subtract(years=2).format("YYYY-MM-DDTHH:mm:ss[Z]")
```

Comment:
Please add comment that date range query param is required for zendesk support api and we could not fetch all data if start date not specified or we should implement it.

Reply:
docstring added